src/fm/cp_fm_diag.F

```diff
@@ -112,15 +123,19 @@ MODULE cp_fm_diag
 !> \param elpa_print logical that determines if information about the ELPA diagonalization should
 !>                   be printed
 !> \param elpa_qr_unsafe logical that enables potentially unsafe ELPA options
+!> \param dlaf_neigvec_min_input ...
 !> \param eps_check_diag_input ...
+!> \par History
+!>      - Add support for DLA-Future (05.09.2023, RMeli)
 !> \author  MI 11.2013
 ! **************************************************************************************************
    SUBROUTINE diag_init(diag_lib, fallback_applied, elpa_kernel, elpa_neigvec_min_input, elpa_qr, &
-                        elpa_print, elpa_qr_unsafe, eps_check_diag_input)
+                        elpa_print, elpa_qr_unsafe, dlaf_neigvec_min_input, eps_check_diag_input)
       CHARACTER(LEN=*), INTENT(IN)                       :: diag_lib
       LOGICAL, INTENT(OUT)                               :: fallback_applied
       INTEGER, INTENT(IN)                                :: elpa_kernel, elpa_neigvec_min_input
       LOGICAL, INTENT(IN)                                :: elpa_qr, elpa_print, elpa_qr_unsafe
+      INTEGER, INTENT(IN)                                :: dlaf_neigvec_min_input
```

Comment:
It seems `dlaf_neigvec_min_input` is unused when `__DLAF` is not set.

Reply:
Thanks. Did you see this anywhere in the public logs, or am I missing something obvious?

Reply:
No, I just noticed from reading the code. Unfortunately with `make -j` the errors are not always within the 100 lines we print. I'll prepare a PR to fix this. In the meantime you can see the entire output by temporarily changing `tail -100` to `cat`  [here](https://github.com/cp2k/cp2k/blob/master/tools/docker/scripts/test_regtest.sh#L59).

Reply:
That's very helpful, thanks!